.vscode/settings.json

```diff
@@ -1,4 +1 @@
-{
-  "solidity.monoRepoSupport": true,
-  "solidity.compileUsingRemoteVersion": "v0.8.21+commit.d9974bed"
-}
+{}
```

Comment:
I left the files in case we wanted to add some defaults but could also remove if you prefer